src/python/data/File.py

```diff
@@ -6,4 +6,8 @@ class File:  # {
     path: str
     matchCount: int
     preview: str
+
+    def __lt__(self, other):  # {
```

Comment:
_ _cmp_ _ ?

Reply:
I think you are old school python developer :D 
`__cmp__` is no longer supported in Python 3, `__lt__` is normal solution now.

Reply:
o-ops. Fine then